FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Tool/Dockerfile

```diff
@@ -1,8 +1,9 @@
-#See https://aka.ms/containerfastmode to understand how Visual Studio uses this Dockerfile to build your images for faster debugging.
-FROM mcr.microsoft.com/dotnet/runtime:6.0 AS base
+FROM mcr.microsoft.com/dotnet/aspnet:6.0 AS base
```

Comment:
Just curious: what's the difference between these two base images?

Reply:
+1

Reply:
The old [dotnet/runtime](https://hub.docker.com/_/microsoft-dotnet-runtime/) image contains .NET runtimes and relates libraries, and the [dotnet/aspnet](https://hub.docker.com/_/microsoft-dotnet-aspnet/) contains ASP.NET Core together with  .NET runtimes and relates libraries.

I find after we add the DicomToDatalake pipeline, running the image built from previous Dockerfile will give an error message that "You must install or update .NET to run this application", I feel some components need more general .NET dependency, so I have som tries, and change the base to [dotnet/aspnet](https://hub.docker.com/_/microsoft-dotnet-aspnet/) solve that issue.

Reply:
Do these two images differ very much in size? It may influence the time of image pulling. By a quick guess I think the dependencies may be introduced by `microsoft.health.dicom.client`. If it is too heavy, we can remove this package by implementing `ChangeFeedEntry` by ourselves.

Reply:
The size of 2 images are similiar, pre offline discussion, we can update the base image to re-use the modules.